pages/common/vault.md

```diff
@@ -0,0 +1,27 @@
+# vault
+
+> HashiCorp Vault, a tool for managing secrets.
+
+- Create a new vault, requiring at least 2 out of 5 keyshares to unseal:
+
+`vault init -key-shares={{5}} -key-threshold={{2}}`
+
+- Unseal a vault, by providing one of the keyshares. Repeat with necessary key-shares until unsealed:
+
+`vault unseal {{key-share-x}}`
+
+- Authenticate client against vault, using an authentication token:
```

Comment:
What precisely is meant by "authenticate client against vault"?

Reply:
HashiCorps Vault is a server or daemon process. You have to authenticate the client process 'vault' before you can do any key-related actions.

Reply:
Ahhh ok